src/cata_utility.h

```diff
@@ -72,6 +74,31 @@ double round_up( double val, unsigned int dp );
  */
 bool isBetween( int test, int down, int up );
 
+/** Sums the two terms, being careful to not trigger overflow or underflow
+ * Correctly handles overflow, underflow, and nonintegral types
+ *
+ * @param a The first addend.
```

Comment:
Is a separate explicit documentation of `a` and `b` really necessary? Isn't it clearly documented and obvious what those two values are via the main function documentation above?